### PR TITLE
Terraform fixes

### PR DESCRIPTION
**Pull request by jamesgreen-moj**
Fixes issues found when trying to deploy development from scratch ( see commit messages for more detail)

### CODE REVIEW

File: modules/vpc/main.tf [hunk 1]
```diff
@@ -13,8 +13,10 @@ module "vpc" {
   manage_default_network_acl    = var.manage_default_network_acl
   manage_default_security_group = var.manage_default_security_group
   manage_default_route_table    = var.manage_default_route_table
-  reuse_nat_ips                 = true
-  external_nat_ip_ids           = aws_eip.gw.*.id
+  reuse_nat_ips                 = terraform.workspace == "development" || terraform.workspace == "pre-production" ? false : true
+  external_nat_ip_ids           = terraform.workspace == "development" || terraform.workspace == "pre-production" ? [] : aws_eip.gw.*.id
+  // Lower costs, by lowering availability
+  single_nat_gateway = terraform.workspace == "development" || terraform.workspace == "pre-production" ? true : false
```
Comment (smjmoj) [inline modules/vpc/main.tf:19]: this variable "single_nat_gateway" is passed into the module "vpc", but isn't used anywhere.

Comment (jamesgreen-moj) [reply modules/vpc/main.tf:19]: It is used here: 
https://github.com/terraform-aws-modules/terraform-aws-vpc/blob/fbd4ff646b4caaa6fcc1fb71bc88d377cc8b3b48/variables.tf#L1183C1-L1183C32

https://github.com/ministryofjustice/nvvs-devops-monitor/blob/7eecd705e972293aacc2c78ff27a7ac0de921da5/modules/vpc/main.tf#L2

Comment (smjmoj) [reply modules/vpc/main.tf:19]: Fine, thanks for the clarification.

**Review by smjmoj — APPROVED** [review]
LGTM

**Pull request merged**